### PR TITLE
Add versionchanged in create_datagram_endpoint doc

### PR DESCRIPTION
**Pull request by romuald**
Most of the parameters were added in 3.4.4 (b9bf913ab32), but this
change was not documented

### CODE REVIEW

**Comment by asvetlov** [comment]
Thanks for contribution but I think the ship has sailed too long ago for such change.
Pythons 3.4 and 3.5 are in security mode, no need to pollute docs with mentioning outdated versions.
Or, at least, it should be 3.5, not a number of bugfix release. For most asyncio users it doesn't matter.

@1st1 what is your opinion?

**Comment by romuald** [comment]
Note that in my case, I've just discovered this after deploying on a Debian jessie, that is still shipping with 3.4.2

**Comment by larryhastings** [comment]
Documentation changes are always fine, even in security-fixes-only branches.  I'll accept this in 3.4 and 3.5 after a review.

File: Doc/library/asyncio-eventloop.rst [hunk 1]
```diff
@@ -390,6 +390,9 @@ Creating connections
    See :ref:`UDP echo client protocol <asyncio-udp-echo-client-protocol>` and
    :ref:`UDP echo server protocol <asyncio-udp-echo-server-protocol>` examples.
 
+   .. versionchanged:: 3.4.4
+
+      The *family*, *proto*, *flags*, *reuse_address*, *reuse_port, *allow_broadcast*, and *sock* parameters were added
```
Comment (1st1) [inline Doc/library/asyncio-eventloop.rst]: Please break the line at 72 characters.

**Review by asvetlov — APPROVED** [review]

**Pull request merged**